Misc/NEWS.d/next/Library/2023-11-24-23-40-00.gh-issue-107361.v54gh46.rst

```diff
@@ -0,0 +1,2 @@
+Add :data:`ssl.VERIFY_X509_PARTIAL_CHAIN` and :data:`VERIFY_X509_STRICT`
+to the default SSL context created with :func:`ssl.create_default_context`.
```

Comment:
Please also add a mention in the "What's new" document under "Summary – Release highlights / Important deprecations, removals or restrictions:", since this is a potentially breaking behavior change which is rather hidden and will not necessarily surface without direct testing in a production environment (it's well possible that local and CI tests work fine, but when you deploy to a particular internal production environment, things start to fail due to different CA/certificate setups).

Reply:
Will do, thank you!

Reply:
2aceb819bed03ba6afedf05bafd7ecd908b0f9e7